### ISSUE TITLE
Searching for material on MSB's site takes time

### ISSUE THREAD

**Issue by dbosk** [OpenSecEd/ism#26]
MSB apparently has several versions of the material available at the same time in different places on the website (informationssäkerhet.se).

Can we make the references more concrete or just include the PDFs in the instruction? It's Creative Commons, so that should work. But then it's the updates ...

**Comment by dbosk** [comment]
The students told me that they spent most of time of the assignment trying to find the exact document, rather than on the assignment itself.

**Comment by lenfra** [comment]
I interpret all of this as, that we need to update the reading instructions. But I don't think we should include PDFs in the instructions, nor be very precise about what exactly they need to read to do the assignment, similar details as before, just updated to the latest version. Otherwise it would just be a disservice to the students, since part of the module is that they should get an holistic view of the process.

**Comment by dbosk** [comment]
Yes, it was mostly that one version has something the other hasn't.
